robocop/config.py

```diff
@@ -2,6 +2,7 @@
 import argparse
 import re
 import fnmatch
+from pathlib import Path
```

Comment:
I think specific module imports should be located under general import instructions

Reply:
Actually I've checked and this import statement was no longer used - I already pass Path() object to config so I didn't need to create new Path() object. Removed the import